0.10/s2i/bin/assemble

```diff
@@ -47,8 +47,20 @@ if [ -n "$NPM_MIRROR" ]; then
 	npm config set registry $NPM_MIRROR
 fi
 
+if [ ! -z $NPM_LOG_LEVEL && $NPM_LOG_LEVEL == "info" ]; then
+	echo "--> ${NPM_LOG_LEVEL} set. Setting npm logging level to 'info'"
+	npm config set loglevel info
+
+elif [ ! -z $NPM_LOG_LEVEL && $NPM_LOG_LEVEL == "silent" ]; then
+	echo "--> ${NPM_LOG_LEVEL} set. Setting npm logging level to 'silent'"
+	npm config set loglevel silent
+else
+	echo "--> No NPM_LOG_LEVEL environment variable set. Setting npm logging level to default 'warn'"
+	npm config set loglevel warn
+fi
```

Comment:
I think you can replace all this if..else with a single block:

```bash
echo "--> Setting NPM loglevel to ${NPM_LOG_LEVEL:-warn}"
npm config set loglevel ${NPM_LOG_LEVEL:-warn}
```

?

And since this affects calls to the `npm` command, I think this should probably be done earlier in the top of the script?